book/website/reproducible-research/rdm/rdm-metadata.md

```diff
@@ -3,15 +3,22 @@
 
 Having data available is of no use if it cannot be understood. 
 For example, a table of numbers is useless if there are no headings that describe what the columns/rows contain.
-Therefore you should ensure that open datasets include consistent core metadata and that the data is fully described.
+Therefore you should ensure that open datasets include consistent core metadata, information about the data, so that the data is fully described.
```

Comment:
```suggestion
Therefore you should ensure that open datasets include consistent core metadata, that is information about the data, so that the data is fully described.
```
I do not follow what it is meant. Adding a "that is" that would make it clearer for me, but I am not sure this is the meaning intended :-)

Reply:
Not entirely, so I'm going to rephrase it so that it is hopefully clear what it means! Thank you!